src/proof/proof_output_channel.h

```diff
@@ -18,28 +18,34 @@
 
 #include <unordered_set>
 
+#include "expr/node.h"
+#include "util/proof.h"
 #include "theory/output_channel.h"
 
 namespace CVC4 {
 
 class ProofOutputChannel : public theory::OutputChannel {
 public:
+  ProofOutputChannel();
+  virtual ~ProofOutputChannel() {}
+
+  void conflict(TNode n, Proof* pf = nullptr) override;
+  bool propagate(TNode x) override;
+  theory::LemmaStatus lemma(TNode n, ProofRule rule, bool, bool, bool) override;
+  theory::LemmaStatus splitLemma(TNode, bool) override;
+  void requirePhase(TNode n, bool b) override;
+  bool flipDecision() override;
+  void setIncomplete() override;
+
+  Node getLastConflict() const { return d_conflict; }
+  Node getLastLemma() const { return d_lemma; }
+
+  Proof* mutable_proof();
```

Comment:
This should be consistent with "getConflict()". It would be good to know if we could strengthen the contract of lemma to also be exactly once. I'll add Andy as a reviewer so he can answer this.

In general, I would prefer snake_case for accessors. But we can have this discussion later.